### DISCUSSION TITLE
I need help building branch l10n_master

### DISCUSSION THREAD

**Discussion by ojenning** [jjazzboss/JJazzLab#168] [Q&A]
Hello,

I am new to Netbeans. It looks like an encoding problem.
For example Bundle_de_DE.properties looks like this:

![Bildschirmfoto_2020-12-30_14-49-02](https://user-images.githubusercontent.com/43676175/103355332-43f7a480-4aae-11eb-8f9e-b15a17a95323.png)

The result is:
![Bildschirmfoto_2020-12-30_14-54-06](https://user-images.githubusercontent.com/43676175/103355621-f6c80280-4aae-11eb-9c79-7c2ac83d4609.png)

The Bundle_* files are stored with UTF-8 encoding.
How do I force Netbeans to handle the UTF-8 files as UTF-8?
I translate a few  passages and now I want to look if the translation is fine in the programs context.

Kind regards
Ole

**Comment by jjazzboss** [comment]
Hi Ole,

Thanks for helping with the translation!

You’re right there is some issues with encoding due to Netbeans. I had to patch Netbeans to make it work. I’m traveling right now but I’ll send you instructions how to do tonight or tomorrow. 

See you later 

Jérôme 

Envoyé de mon iPhone

**Comment by ojenning** [comment]
Hello Jérôme,

that is fine. Don't hurry.
Even in this small "Options"-Screenshot we have to change some  passages.

Have a nice trip
Ole

**Comment by jjazzboss** [comment]
I added a explanation page how to test JJazzLab with latest translations, see this [page](https://jjazzlab.gitbook.io/user-guide/contribute/translate-jjazzlab/testing-translations).

**Reply by ojenning** [reply]
Hello Jérôme,

many tanks!
That helps a lot!
